src/trackHttp.test.js

```diff
@@ -0,0 +1,85 @@
+import {trackPromise, emitter} from './trackHttp';
+import {Emitter} from './tinyemmiter';
+
+
+describe('trackPromise', () => {
+  test('On Initial case, promise fired, promise emitter.emit is called', () => {
+    // Arrange
+    emitter.emit = jest.fn((a,b) => {
+      return;
+    });
+
+    const myPromise =  Promise.resolve().then(() => {
```

Comment:
The idea here is that this promise is not going to be resolved by the time the "expect" clause is evaluated. Thus, I would rename this to myPendingPromise or myDeferredPromise for better clarity. 
(On a side note, I really miss the deprecated Promise.defer() method here, it feels rather artificial and convoluted to do it this way, but I cannot think of a better way either).